debian/changelog

```diff
@@ -1,3 +1,18 @@
+gnome-control-center (1:3.36.0-1endless0) unstable; urgency=medium
```

Comment:
I think technically this should be `1:3.36.0-1endless1`, since numbering of released versions should start from 1?

Reply:
On a quick look, the wiki said  `endless0` : https://phabricator.endlessm.com/w/howto/rebase_debian_packages/#ask-someone-to-review-th

Reply:
No, I’m probably missing something. Go with `endless0`.